main.py

```diff
@@ -0,0 +1,48 @@
+from gym_jadx.envs.jadx_env import JadxEnv
+import pygame
+import numpy as np
+import time
+import cv2
+
+env = JadxEnv()
+DISPLAY_SIZE = (1280, 720)
+ENV_SIZE = (env.width, env.height)
+WIDTH_RATIO = DISPLAY_SIZE[0] / ENV_SIZE[0]
+HEIGHT_RATIO = DISPLAY_SIZE[1] / ENV_SIZE[1]
+
+pygame.init()
+display = pygame.display.set_mode(DISPLAY_SIZE)
+resized = cv2.resize(env.frame_buffer, dsize=(DISPLAY_SIZE[1], DISPLAY_SIZE[0]))
+pygame.surfarray.blit_array(display, resized)
+pygame.display.update()
+
+running = True
+while running:
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            running = False
+        if event.type == pygame.MOUSEBUTTONDOWN:
+            # Left click
+            if event.button == 1:
+                point = np.array([event.pos[0] / WIDTH_RATIO, event.pos[1] / HEIGHT_RATIO])
```

Comment:
I would find it better if this was inside `env.step()`. Users of the env should not have to deal with how to recalculate the position of the click. Maybe put the relevant display size parameters into the environment and then the ratios and the conversion of the point can be calculated there.

Reply:
I did this after Daniel wanted me to scale the pygame only, without changing the size of the environment. Afaik this will be used just for the presentation as well. I could do it inside the environment, but then the step function should either output the scaled image (I think this is not wanted) or it should output both the scaled and the original
image. To keep it simple I did it in the main function, but since it is just for presenting it can be ignored.